src/TestMapper.php

```diff
@@ -28,23 +31,29 @@ public function getMap(): array {
       $className = (string) $class->attributes()['name'];
       try {
         $reflection = new \ReflectionClass($className);
-      }
-      catch (\ReflectionException $e) {
+      } catch (\ReflectionException $e) {
         // Couldn't find the class.
         continue;
       }
       $filename = $reflection->getFileName();
+      $filename = \str_replace($this->prefix, '', $filename);
```

Comment:
We should make sure this is at the start of the string, prefix could be something like `/` or `/src/` which could be problematic.

```suggestion
      if (\str_starts_with($filename, $this->prefix)) {
        $filename = \substr($filename, \strlen($this->prefix));
      }
```